packages/analytics-js/src/components/errorListener/windowUnhandledRejection.ts

```diff
@@ -0,0 +1,53 @@
+import type { IPluginsManager } from '@rudderstack/analytics-js-common/types/PluginsManager';
+import type { ILogger } from '@rudderstack/analytics-js-common/types/Logger';
+import { state } from '../../state';
+import type { ErrorMetaData } from './types';
+
+const attachUnhandledRejection = (pluginsManager: IPluginsManager, logger?: ILogger) => {
+  const listener = (evt: any) => {
+    let error = evt.reason;
+    let isBluebird = false;
```

Comment:
What is this?

Reply:
https://apps.microsoft.com/detail/9PNXW61T4T0V?hl=en-US&gl=US bluebird is a browser. do we have an issue specificaly with it?